webroot/js/participantSummary.js

```diff
@@ -0,0 +1,129 @@
+let siteList;
+
+/* ***** multi-use error handler *** */
+function participantSummaryErrorHandler(err, divContainer) {
+    const errorMessage = prettifyJson(err.responseText);
+    $(divContainer).html(errorMessage).show();
+};
+
+function loadAllSitesCallback(result) {
+    siteList = JSON.parse(result).map(function(item) { return { name: item.name, id: item.id } });
```

Comment:
nit: is there a good reason to use this notation over arrow notation? e.g.
```
siteList = JSON.parse(result).map((item) => { return { name: item.name, id: item.id } });
```
I think arrow notation is easier to read

Reply:
Yeah, I originally used the old notation just for consistency.  But since this is a new file I think you're right.  I changed them all.